### DISCUSSION TITLE
Ensure all code examples in docs can actually run

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#207] [Ideas]
Many of the examples in our `docs/` folder have runtime errors, which makes them less useful in the [Radashi Playground](https://radashi.js.org/playground/). For context, the playground is generated from the **first example** of each function's documentation. Currently, other examples are not included.

### Offending pages
- `docs/array/list.mdx` (undeclared variable)
- `docs/async/defer.mdx` (undeclared variable)
- `docs/async/guard.mdx` (missing import)
- `docs/async/map.mdx` (undeclared variable)
- `docs/async/reduce.mdx` (undeclared variable)
- `docs/async/retry.mdx` (undeclared variable)
- `docs/async/tryit.mdx` (undeclared variable)
- `docs/async/withResolvers.mdx` (missing import)
- `docs/curry/debounce.mdx` (undeclared variable)
- `docs/curry/once.mdx` (missing import)
- `docs/curry/throttle.mdx` (missing import)
- `docs/function/castComparator.mdx` (incorrect import)
- `docs/number/clamp.mdx` (invalid call)
- `docs/object/cloneDeep.mdx` (infinite recursion)
- `docs/typed/isResultErr.mdx` (missing import)
- `docs/typed/isResultOk.mdx` (missing import)

**Comment by MarlonPassos-git** [comment]
https://github.com/radashi-org/radashi/pull/247
